datahub/company/test/test_company_views.py

```diff
@@ -1420,7 +1420,7 @@ class TestAddCompany(APITestMixin):
     """Tests for adding a company."""
 
     @pytest.mark.parametrize(
-        'data,expected_response',
+        'data, expected_response, feature_flags',
```

Comment:
```suggestion
        'data,expected_response,feature_flags',
```
Pytest used to only allow for this first parameter to [`pytest.mark.parameterize`](https://docs.pytest.org/en/6.2.x/reference.html#pytest.python.Metafunc.parametrize) to be separated by `,` and not `, `. If the function handles spaces in the string in the desired way right now then that's cool, but the fact it's not expressly included in the linked documentation makes me think that it's implicit functionality that might be removed in a future release. Plus the other usages of this function in this module don't have spaces in the equivalent argument, so it's good to keep the same conventions